# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pyzor/fixinterpreter.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN pyzor-0.4.0.orig/scripts/pyzor pyzor-0.4.0/scripts/pyzor
--- pyzor-0.4.0.orig/scripts/pyzor	2002-06-08 19:24:20.000000000 -0400
+++ pyzor-0.4.0/scripts/pyzor	2003-05-12 11:09:12.000000000 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import pyzor.client
 pyzor.client.run()
diff -urN pyzor-0.4.0.orig/scripts/pyzord pyzor-0.4.0/scripts/pyzord
--- pyzor-0.4.0.orig/scripts/pyzord	2002-09-07 23:33:44.000000000 -0400
+++ pyzor-0.4.0/scripts/pyzord	2003-05-12 11:09:17.000000000 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import os
 import os.path

